### ISSUE TITLE
Python 3 type hints

### ISSUE THREAD

**Issue by cerealkill** [NiklasRosenstein/pydoc-markdown#101]
Hi Niklas,

First, thanks for working on this tool, I am really enjoying to use it. So, I updated from version 2.0.5 to 2.1.0 because I wanted to use parsed args and google Preprocessor, but I noticed that type hints are not detected, like so:
```
  File "<unknown>", line 1
    Client(in3_config: in3.model.client.Config = None)
                     ^
SyntaxError: invalid syntax
```
If I remove the type hint in my code, then it works. This feature was working on the previous version.

I am using python 3.7.

BR
Paul

**Comment by NiklasRosenstein** [comment]
Hey @cerealkill ,

Thanks for reporting. This should be fixed in 2.1.1. 

Related issue: #100

**Comment by cerealkill** [comment]
Thanks a bunch!
